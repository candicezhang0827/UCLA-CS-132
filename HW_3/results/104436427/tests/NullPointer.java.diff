1,3c1,2
< --- Error (ProgramFault) ---
< use of uinitialized value
<   Main() line 4
---
> --- Error (Explicit) ---
> null pointer
