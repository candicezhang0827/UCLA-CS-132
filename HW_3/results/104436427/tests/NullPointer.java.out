--- Error (ProgramFault) ---
use of uinitialized value
  Main() line 4
